azure-kit-demo/src/main/java/com/vaadin/azure/demo/Application.java

```diff
@@ -0,0 +1,29 @@
+package com.vaadin.azure.demo;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import com.vaadin.flow.component.dependency.NpmPackage;
+import com.vaadin.flow.component.page.AppShellConfigurator;
+import com.vaadin.flow.server.PWA;
+import com.vaadin.flow.theme.Theme;
+
+/**
+ * The entry point of the Spring Boot application.
+ *
+ * Use the @PWA annotation make the application installable on phones, tablets
+ * and some desktop browsers.
+ *
+ */
+@SpringBootApplication
+@Theme(value = "azure-kit-demo")
+@PWA(name = "Azure Kit Demo", shortName = "Azure Kit Demo", offlineResources = {})
+@NpmPackage(value = "line-awesome", version = "1.3.0")
+@NpmPackage(value = "@vaadin-component-factory/vcf-nav", version = "1.0.6")
+public class Application implements AppShellConfigurator {
```

Comment:
What about adding `@Push` annotation to be able to check that web socket works seamlessly on the application deployed on the cluster?